experimental/dds/tree/src/test/utilities/IdCompressorTestUtilities.ts

```diff
@@ -297,7 +297,8 @@ export class IdCompressorTestNetwork {
 		numIds: number,
 		overrides: { [index: number]: string } = {}
 	): OpSpaceCompressedId[] | IdCreationRange {
-		assert(numIds > 0, 'Must allocate a non-zero number of IDs');
+		// Must allocate a non-zero number of IDs
+		assert(numIds > 0);
```

Comment:
any reason for these changes? seems like the previous version would give slightly clearer errors.

Reply:
it would get a short code, then fail to compile due to type mismatch on the second argument.